client/src/components/UserCard.tsx

```diff
@@ -28,11 +29,13 @@ export default function UserCard(props: Props) {
   const status = props.item != null ? props.item.status : 'good';
   const isSandbox = PLAID_ENV === 'sandbox';
   const isGoodState = status === 'good';
+  const isAuth = IS_PROCESSOR === 'true' ? false : true;
+  const isIdentity = props.user.should_verify_identity ? true : false;
```

Comment:
you can probably just remove the ternary operator here and just have `const isIdentity = props.user.should_verify_identity`